regexes/device/mobiles.yml

```diff
@@ -2380,6 +2410,14 @@ Datsun:
     - regex: 'DATSUN_D5500'
       model: 'D5500'
 
+# Dapeng
+Dapeng:
+  regex: 'A75 Build'
```

Comment:
Not sure if that detection doesn't match to many things. Actually there are other devices named A75, like https://www.oppo.com/en/smartphone-a75/ or one from Micromax. Guess we shouldn't add it if it isn't unambiguous?

Reply:
Remove.